src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -248,8 +240,7 @@ public Camera GetCamera(uint index)
         /// </summary>
         /// <param name="name"> string keyword of Camera to be retrieved.</param>
         /// <returns>Camera that has the name as a View.Name property</returns>
```

Comment:
at line 239 - there is no index
```suggestion
        /// <returns>Camera that has View.Name property value equal to <paramref name="name"/>.</returns>
```

Reply:
Oh. Thank you.